src/server/server_family.cc

```diff
@@ -2520,19 +2610,27 @@ void ServerFamily::Role(CmdArgList args, ConnectionContext* cntx) {
 
   } else {
     unique_lock lk{replicaof_mu_};
-    Replica::Info rinfo = replica_->GetInfo();
-    rb->StartArray(4);
+
+    size_t additional_replication = cluster_replicas_.size();
```

Comment:
I'd drop `additional_replication` and use the `size()` method inline, I don't think it adds clarity